tests/net/all/prj.conf

```diff
@@ -106,6 +107,7 @@ CONFIG_NET_CONTEXT_SYNC_RECV=y
 CONFIG_NET_CONTEXT_CHECK=y
 
 # SLIP
+CONFIG_NET_SLIP_TAP=y
```

Comment:
why do you need this? Now it is set automatically for those boards that need it, no need to add it here anymore.